lib/mhv_logging/client.rb

```diff
@@ -9,71 +9,24 @@
 require 'rx/configuration'
 require 'rx/client_session'
 
-# NOTE: This client uses the exact same configuration, breakers, and client session
+# NOTE: This client uses the exact same configuration and client session
 # as Rx does. It is the same server, but we're building it as a separate client
 # for better separation of concerns since it can be invoked when using secure messaging
 
 module MHVLogging
   # Core class responsible for api interface operations
-  class Client
+  class Client < Common::Client::Base
+    include Common::Client::MHVSessionBasedClient
     include MHVLogging::API::Audits
     include Rx::API::Sessions
```

Comment:
I might keep Rx::API::Sessions as a concern -- since it's shared between MHVAudit and Rx.